src/sentry/api/endpoints/avatar/user.py

```diff
@@ -48,4 +48,6 @@ def put(self, request: Request, **kwargs) -> Response:
             user_id=request.user.id,
             attrs=dict(avatar_url=avatar_url, avatar_type=user_avatar.avatar_type),
         )
-        return Response(serialized_user)
+        if serialized_user:
```

Comment:
Addressing a general concern where before we were not handling the `None` case better.  See https://github.com/getsentry/sentry/pull/55424/files#diff-d56e94da1a1f840fadade0926071161991dd600c4fff3d2eb4903f7a4ba516aeR142

Reply:
This is sort of a "problem" with `Any` type annotations.  I wonder if there is a way to make an `Any | None` that still enforces a  `None` check on the caller?